common/src/main/java/org/broadleafcommerce/common/web/BroadleafRequestProcessor.java

```diff
@@ -66,6 +65,7 @@ public class BroadleafRequestProcessor extends AbstractBroadleafWebRequestProces
     public static final String SITE_DISABLE_SANDBOX_PREVIEW = "site.disable.sandbox.preview";
 
     private static final String SANDBOX_ID_PARAM = "blSandboxId";
+    private static final String INCLUDE_MY_CHANGES_VAR = "blIncludeMyChanges";
```

Comment:
I think we can delete this and use BroadleafIncludeMyChangesResolver.INCLUDE_MY_CHANGES_VAR